lib/dradis/plugins/projects/upload/v1/template.rb

```diff
@@ -55,6 +55,19 @@ def create_activity(trackable, xml_activity)
         validate_and_save(activity)
       end
 
+      def create_issue(issue, xml_issue)
```

Comment:
what is the purpose of this change in the old template?

Reply:
I wanted to have a method that I can override in v2. Of course, I can just override the original method `parse_issues` but it seemed excessive.